test/fixtures/publishers.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  type: 
+
+two:
+  name: MyString
+  type: 
```

Comment:
My policy recently with these generated files is to discard them if we're not actively using them.